bf2fc6cc711aee1a0c2a/kas-fleet-manager
# chore: generate openapi

## Description
New /federate endpoint has now been added to the KAS Fleet Manager OpenAPI spec. The OpenAPI needs to be re-generated so that the docs in api.stage.openshift.com and api.openshift.com gets updated with this new endpoint.

Also updated examples of error responses so that its id matches the ServiceError code as expected.

## Verification Steps
- CI Passing
- Run make run/docs
   - Verify OpenAPI docs are as expected (/federate endpoint should be available, all example error responses should have its id match its service error code).

## Checklist (Definition of Done)
